examples/07-EMIT/interference_gui.py

```diff
@@ -204,25 +242,50 @@ def open_file_dialog(self):
             self.file_path_box.setText(fname[0])
 
             # Close previous project and open specified one
-            emitapp.close_project()       
-            emitapp.load_project(self.file_path_box.text())
+            if self.emitapp is not None:
+                self.emitapp.close_project()
+            desktop_proj = desktop.load_project(self.file_path_box.text())
 
             # Check if project is already open
-            if emitapp.lock_file == None:
+            if desktop_proj.lock_file == None:
                 msg = QtWidgets.QMessageBox()
                 msg.setWindowTitle("Error: Project already open")
                 msg.setText("Project is locked. Close or remove the lock before proceeding. See AEDT log for more information.")
                 x = msg.exec()
                 return
             
             # Populate design dropdown with all design names
-            designs = emitapp.oproject.GetDesigns()
-            emit_designs = [d for d in designs if d.GetDesignType() == "EMIT"]
+            designs = desktop_proj.design_list
+            emit_designs = []
+            self.populating_dropdown = True
             self.design_name_dropdown.clear()
-            for d in emit_designs:
-                self.design_name_dropdown.addItem(d.GetName())
-                
-            self.design_name_dropdown.setEnabled(True)
+            self.populating_dropdown = False
+            for d in designs:
+                design_type = get_pyaedt_app(desktop_proj.project_name, d).design_type
```

Comment:
Is this pretty fast? I'm not sure if there is much overhead creating an App object, but there is some for EMIT. Can we avoid creating the app objects by using desktop.design_type()?

Reply:
@myoung301 Unfortunately I don't think so. From what I saw, `desktop.design_type()` just returns the design type for the active design. I couldn't find a way to get the design type based on the design names. (doesn't mean there isn't something I missed)

Reply:
Here's the other PR: https://github.com/ansys/example-data/pull/188

Reply:
@jsalant22 - The function says it takes a project name and design name and seems to imply it can return the type for non-active projects and designs:
https://github.com/ansys/pyaedt/blob/7b96bc5df97390231df4ef118f0fd687948da83b/pyaedt/desktop.py#L1254-L1264